### PR TITLE
Fix listener map type error

### PR DESCRIPTION
**Pull request by bpilot**
This will resolve issue #649 so Closure is even better for those using type annotations.

### CODE REVIEW

File: closure/goog/deps.js [hunk 1]
```diff
@@ -1,31 +1,5 @@
...
-goog.addDependency('../../third_party/closure/goog/dojo/dom/query.js', ['goog.dom.query'], ['goog.array', 'goog.dom', 'goog.functions', 'goog.string', 'goog.userAgent'], false);
-goog.addDependency('../../third_party/closure/goog/loremipsum/text/loremipsum.js', ['goog.text.LoremIpsum'], ['goog.array', 'goog.math', 'goog.string', 'goog.structs.Map', 'goog.structs.Set'], false);
-goog.addDependency('../../third_party/closure/goog/mochikit/async/deferred.js', ['goog.async.Deferred', 'goog.async.Deferred.AlreadyCalledError', 'goog.async.Deferred.CanceledError'], ['goog.Promise', 'goog.Thenable', 'goog.array', 'goog.asserts', 'goog.debug.Error'], false);
-goog.addDependency('../../third_party/closure/goog/mochikit/async/deferredlist.js', ['goog.async.DeferredList'], ['goog.async.Deferred'], false);
-goog.addDependency('../../third_party/closure/goog/osapi/osapi.js', ['goog.osapi'], [], false);
-goog.addDependency('../../third_party/closure/goog/svgpan/svgpan.js', ['svgpan.SvgPan'], ['goog.Disposable', 'goog.events', 'goog.events.EventType', 'goog.events.MouseWheelHandler'], false);
+// This file was autogenerated by depswriter.py.
```
Comment (Dominator008) [inline closure/goog/deps.js]: Seems like in this case it's better to manually edit the `deps.js` file. I think changing
`goog.addDependency('events/listenermap.js', ['goog.events.ListenerMap'], ['goog.array', 'goog.events.Listener', 'goog.object'], false);`
to
`goog.addDependency('events/listenermap.js', ['goog.events.ListenerMap'], ['goog.array', 'goog.asserts', 'goog.events.Listener', 'goog.object'], false);`
is the only modification needed.

**Comment by shicks** [comment]
Thanks for putting this together.  Can you please sign the CLA as described by GoogleBot's post above?  Once that's taken care of (and GoogleBot tags this as CLA: yes) then we can merge it.

Also, if you could revert the change to deps.js, that would be great.  We'll take care of regenerating it automatically when we merge.

**Comment by joeltine** [comment]
@bpilot Still waiting on a CLA from you. We can't move unless you sign it.

**Comment by joeltine** [comment]
When/if you sign the CLA, please ping the thread.

**Pull request closed without merging**